scripts/ci/config.yml

```diff
@@ -18,6 +18,11 @@ general_features:
   is_dir: true
   type_tests: "e2e"
   dependencies:
+    - "pytest"
```

Comment:
we don't need these here, everything here besides `Pillow` is already defined in the base test-requirements.txt

Reply:
And that's what we should not do I guess